src/interface/src/styleguide/filter-dropdown/filter-dropdown.component.ts

```diff
@@ -60,7 +60,10 @@ export class FilterDropdownComponent<T> implements OnInit {
    * provided menuItems is not already a string.
    */
   @Input() displayField?: keyof T;
-
+  /**
+   * Dynamically set the width from the consumer
+   */
+  @Input() width: string | number = '200px'; // Default width
```

Comment:
what do you think of instead of letting any arbitrary number, set up literal sizes (small, medium, large, whatever we need)
Granted, wont be as flexible, but way easier to maintain consistency

Reply:
Absolutely. I thought I'd give arbitrary width a try first, but there are a lot of dependent elements that make this harder. I also think it might make sense to offer an optional width for the panel layer separately.